controller/src/enhancements/player.rs

```diff
@@ -130,11 +131,11 @@ impl PlayerESP {
             return Ok(None);
         }
 
-        let controller_handle = player_pawn.m_hController()?;
-        let current_controller = ctx.cs2_entities.get_by_handle(&controller_handle)?;
+        let player_controller_handle = player_pawn.m_hOriginalController()?;
```

Comment:
Why are you using `m_hOriginalController` instead of `m_hController` here?

Reply:
m_bPawnHasDefuser() is on CCSPlayerController, m_hController is CBasePlayerController, but I gess I can cast it to CCSPlayerController or is there a better way how to get m_bPawnHasDefuser()?

Reply:
I'm not quite sure, why `m_hOriginalController` and `m_hController` got different signatures.  
I looked up some other values and I think a proper solution would be:
```
player_pawn.m_pItemServices()?
  .cast::<CCSPlayer_ItemServices>()
  .reference_schema()?
  .m_bHasDefuser()
```
  
I'm quite sure it's save to assume, that the item services of a `C_CSPlayerPawn` would be `CCSPlayer_ItemServices`.  
(I haven't yet tested the solution)